src/common/eeprom.cpp

```diff
@@ -91,6 +91,7 @@ typedef struct _eeprom_vars_t {
     Sheet SHEET_PROFILE7;
     uint32_t SELFTEST_RESULT;
     uint8_t QR_PRIVACY;
+    uint8_t FAN_CHECK;
```

Comment:
/suggestion/ Consider renaming it to `FAN_CHECK_ENABLED` to make It clear that it stores a boolean